base/docs/helpdb/Base.jl

```diff
@@ -8089,7 +8089,7 @@ fld1
 """
     mod1(x, y)
 
-Modulus after flooring division, returning a value in the range `(0, y]`.
+Modulus after flooring division, returning a value `r` such that `mod(r, y)==mod(x,y)` in the range `(0, y]` for positive `y` and in the range `[y,0)` for negative `y`.
```

Comment:
still needs wrapping, and should be consistent - if `mod(r, y)` has a space after the comma then `mod(x,y)` should too (or neither), and the ranges should be math quoted here too